CompetitionTemplate/src/competition/subsystems/collector/commands/UpdateCollectorSensorsCommand.java

```diff
@@ -0,0 +1,29 @@
+package competition.subsystems.collector.commands;
+
+import com.google.inject.Inject;
+
+import competition.subsystems.arm.ArmSubsystem;
+import competition.subsystems.collector.CollectorSubsystem;
+import competition.subsystems.hanger.HookSubsystem;
+import xbot.common.command.BaseCommand;
+
+public class UpdateCollectorSensorsCommand extends BaseCommand {
```

Comment:
+1 for good naming - it should be a separate PR, since we had already normalized on "UpdateSensors" across many subsystems.